src/NetMQ.Tests/NetMQMonitorTests.cs

```diff
@@ -188,5 +188,86 @@ public void ConvertArgThrowsForInvalidType()
             MonitorEvent monitorEvent = new MonitorEvent(SocketEvents.All, addr: "", arg: socket!);
             Assert.Throws<ArgumentException>(() => monitorEvent.ConvertArg<int>());
         }
+
+        [Fact]
+        public void INetMQMonitorCanBeMocked()
```

Comment:
This test doesn't really add any value. It's just testing the mock that it added. Remove the test and mock.